pkg/providers/imagefamily/bootstrap/aksbootstrap.go

```diff
@@ -437,6 +437,9 @@ func CredentialProviderURL(kubernetesVersion, arch string) string {
 		credentialProviderVersion = "1.29.2"
 	case 30:
 		credentialProviderVersion = "1.30.0"
+
+	case 31:
+		credentialProviderVersion = "1.31.0"
```

Comment:
I presume you checked there is one (for both amd and arm), and it will be supported?